src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/CreateMethodologyForSpecificPublicationAuthorizationHandlers.cs

```diff
@@ -4,7 +4,6 @@
 using GovUk.Education.ExploreEducationStatistics.Common.Services.Security;
 using GovUk.Education.ExploreEducationStatistics.Content.Model;
 using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
-using IdentityServer4.Extensions;
```

Comment:
Just for reference I have a change on my current branch to remove all other usages of `NullOrEmpty` in this package and I replace them with our own version of it.

Reply:
OK!